runtests.py

```diff
@@ -2178,6 +2197,9 @@ def runtests(options, cmd_args, coverage=None):
             sys.stderr.write("Including CPython regression tests in %s\n" % sys_pyregr_dir)
             test_suite.addTest(filetests.handle_directory(sys_pyregr_dir, 'pyregr'))
 
+    if options.code_style and options.shard_num <= 0:
+        test_suite.addTest(TestCodeFormat(options.cython_dir))
```

Comment:
I assume this test is included in the test name selection? Meaning, when I pass a name pattern, it will not run, unless selected by the pattern? (That would be important, since I don't want to disable code style analysis each time I run a specific test.)

Reply:
I see what you mean. I'll dig to see how it can be done.

Reply:
Solved.